client/components/button/button.css

```diff
@@ -1,5 +1,74 @@
-.unique-button-class {
-  color: white;
+@import '../typography/typography.css';
+@import '../typography/color.css';
+
+button {
+  padding: 0;
   border: none;
-  background-color: red;
+  font: inherit;
+  color: inherit;
+  background-color: transparent;
+  outline: none;
 }
+
+.button {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 10px 16px;
+  height: 48px;
+  border-radius: 8px;
+  line-height: 28px;
```

Comment:
The same line-height is defining in typography heading-h5

Reply:
Yes. I've corrected it. Thanks.